build/int.cloudbuild.yaml

```diff
@@ -23,19 +23,19 @@ steps:
   - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
 - id: create
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
-  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do create']
+  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && source_test_env && init_credentials && cd test/integration && RUN_STAGE=init go test -v ./... -p 1']
```

Comment:
I believe we  can just do `source_test_env && init_credentials` alongside `prepare_environment` and it should stick

Reply:
I'm going to stick with this for now. Once we have the CLI that will be better anyways.